src/metabase/lib/js.cljs

```diff
@@ -456,3 +456,25 @@
   filtered out based on values of the LHS or RHS, but in the future we can add this -- see #31174."
   [a-query stage-number lhs-column-or-nil rhs-column-or-nil]
   (to-array (lib.core/join-condition-operators a-query stage-number lhs-column-or-nil rhs-column-or-nil)))
+
+(defn ^:export expression
+  "Adds an expression to query."
+  ([a-query expression-name an-expression-clause]
+   (expression a-query -1 expression-name an-expression-clause))
+  ([a-query stage-number expression-name an-expression-clause]
+   (lib.core/expression a-query stage-number expression-name an-expression-clause)))
+
+(defn ^:export expressions
+  "Get the expressions map from a given stage of a `query`."
+  ([a-query]
+   (expressions a-query -1))
```

Comment:
I think Anton made some changes so the `stage-number` is always required in the JS wrappers, so there's not really a reason to add the arities without `stage-number` in `metabase.lib.js` anymore, since they're going to go unused. Going forward we can just add the arities with explicit stage numbers